### DISCUSSION TITLE
Can we change an icon based on tab title alone?

### DISCUSSION THREAD

**Discussion by MarkTheTestStrategist** [furybee/chrome-tab-modifier#368] [General]
Hi,

Is there some way to change the tabs icon, based on the tabs title? 

I am wanting to add a bug icon to bug tickets that are open in a tab in azure devops. 
As we use Azure devops for a lot of things, the url is not necessarily unique.

The title however persists Bug before the bug id. 

![image](https://github.com/user-attachments/assets/68b8869d-5160-4c38-8a52-b126725cb7ec)

I noticed in the advanced section, there appears to be an option for this and I have entered `Bug*` but it changes all tabs that begin with `https://dev.azure.com/<workspace>/` as you'd expect when using a url.

I can't remove the url as it disables the advanced section.

**Comment by sylouuu** [comment]
Sorry @GenXPhooey it's not doable or planned at the moment.

Bests
